js/mexc.js

```diff
@@ -1848,11 +1848,11 @@ module.exports = class mexc extends Exchange {
         if (orderType === 'LIMIT') {
             orderType = 'LIMIT_ORDER';
         }
-        const postOnly = this.safeValue (params, 'postOnly', false);
-        const timeInForce = this.safeString (params, 'timeInForce');
-        const maker = (postOnly || (timeInForce === 'PO'));
+        let timeInForce = this.safeString (params, 'timeInForce');
+        let postOnly = false;
+        [ type, postOnly, timeInForce, params ] = this.isPostOnly (type, timeInForce, undefined, params);
```

Comment:
This could probably be something like

```
let defaultTimeInForce = this.safeString (params, 'timeInForce');
const [ orderType, postOnly, timeInForce, query ] = this.isPostOnly (type, defaultTimeInForce, type === 'POST_ONLY', params);
```

Reply:
You should definately add the `upperCaseType === 'POST_ONLY'` as the third argument, that one is important for functionality